design_drafts/components_architecture_and_urdf_examples.md

```diff
@@ -71,31 +71,32 @@ Note:
     </hardware>
     <joint name="joint1">
       <classType>ros2_control_components/MultiInterfaceJoint</classType>
-      <commandInterfaceType>position</commandInterfaceType>
-      <commandInterfaceType>velocity</commandInterfaceType>
-      <commandInterfaceType>effort</commandInterfaceType>
+      <commandInterfaceType name="position">
```

Comment:
I would rather prefer the following:
`<commandInterfaceType name="position" min="-1" max="1">`

Nevertheless, your approach is better if using interface type is not defined.

Reply:
let's go for this one.